src/Adapter/Container/DoctrineBuilderExtension.php

```diff
@@ -40,25 +41,30 @@
  */
 class DoctrineBuilderExtension implements ContainerBuilderExtensionInterface
 {
-    /** @var string */
-    private $cacheDir;
+    /** @var EnvironmentInterface */
+    private $environment;
 
     /**
-     * @param string $cacheDir
+     * @param EnvironmentInterface $environment
      */
-    public function __construct($cacheDir)
+    public function __construct(EnvironmentInterface $environment)
     {
-        $this->cacheDir = $cacheDir;
+        $this->environment = $environment;
     }
 
     /**
      * {@inheritdoc}
      */
     public function build(ContainerBuilder $container)
     {
-        $yamlParser = new YamlParser($this->cacheDir);
+        $configDirectories = [$container->getParameter('kernel.project_dir') . '/app/config'];
```

Comment:
```suggestion
        $configDirectories = [$container->getParameter('kernel.root_dir') . '/config'];
```
Although this path may be better set up as a parameter instead of hard coded?

Reply:
Good idea for `kernel.root_dir` however I'm not sure it's worth adding a parameter for the config directory, it will remain here by convention So letting the extension compute the path based on the `kernel.root_dir` seems enough to me